frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -688,6 +688,7 @@ To:
     })
     if Device:isKobo() and Device:canToggleChargingLED() then
         table.insert(self.menu_items.developer_options.sub_item_table, {
+            --@translators This is a debug option to help determine cases when standby failed to initiate standby properly. PM = power management.
```

Comment:
@NiLuJe

Reply:
Missed a space after the double dash, but, yeah, lgtm ;).

(Possibly just"when standby failed to initiate properly" to avoid the repetition?)

Reply:
```suggestion
            -- @translators This is a debug option to help determine cases when standby failed to initiate properly. PM = power management.
```